roles/strimzi_kafka/tasks/install.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Install Strimzi Operator
+  kubernetes.core.k8s:
+    state: present
+    definition: "{{ lookup('url', item, split_lines=False) }}"
+  with_items:
+    - "{{ kafka_install_url }}"
+
+- name: Wait for Kafka Deployment
+  kubernetes.core.k8s_info:
+    kind: Deployment
+    wait: yes
+    wait_condition:
+      type: Available
+      status: "True"
+      reason: MinimumReplicasAvailable
+    name: "{{ item }}"
+    namespace: "{{ kafka_namespace }}"
+    wait_timeout: 360
+  with_items:
+    - strimzi-cluster-operator
+
+- name: Create Kafka Cluster
+  kubernetes.core.k8s:
+    state: present
+    namespace: "{{ kafka_namespace }}"
+    template: templates/cluster.yaml.j2
```

Comment:
Main recommended change: would suggest to load the default template as part of `defaults/main.yaml` using `lookup` in the same way we do for the default seldon gateway:
https://github.com/SeldonIO/ansible-k8s-collection/blob/f7d283abd41cffd85c33514e8847b073c54e24e5/roles/istio/defaults/main.yaml#L9

as this will open means to customise it by the users of the role easily by just overwriting the variable.

Reply:
However, we can just do it if the need for customisation arise as we already know how to do it, so no need to adjust in this PR.